distribution/helm/vector-aggregator/templates/service-headless.yaml

```diff
@@ -7,8 +7,6 @@ metadata:
 spec:
   clusterIP: None
   ports:
-    {{- if eq (include "vector-aggregator.servicePortsPresent" .) "true" -}}
```

Comment:
This wrapper should remain to be consistent with the non-headless service.

Reply:
Sure I can put it back in, however, it ends up the same output and removing a branch point reduces cyclomatic complexity.

Reply:
I agree this is not pretty, but we should either use that in both places - merely for consistency - or figure out another way to do it.
What we could do, I guess, is instead of doing
```
{{- if eq (include "vector-aggregator.servicePortsPresent" .) "true" -}}
```
at the `vector/distribution/helm/vector-aggregator/templates/service.yaml`, try something like
```
{{- if not (empty (include "vector-aggregator.servicePorts" .)) -}}
```
and remove the `vector-aggregator.servicePortsPresent` itself.